asyncio_connection_pool/contrib/aioredis.py

```diff
@@ -1,16 +1,18 @@
-import aioredis
 from functools import partial
+
+from redis import asyncio as aioredis
+
 from asyncio_connection_pool import ConnectionStrategy
 
 __all__ = ("RedisConnectionStrategy",)
 
 
-class RedisConnectionStrategy(ConnectionStrategy[aioredis.Redis]):  # type: ignore
+class RedisConnectionStrategy(ConnectionStrategy[aioredis.Redis]):
     def __init__(self, *args, **kwargs):
-        self._create_redis = partial(aioredis.create_redis, *args, **kwargs)
+        self._create_redis = partial(aioredis.Redis, *args, **kwargs)
 
-    async def make_connection(self):
-        return await self._create_redis()
+    async def make_connection(self) -> aioredis.Redis:
+        return self._create_redis()
```

Comment:
This probably doesn't actually allocate a connection since redis-py manages connections internally. If there's a public API for explicitly managing connections maybe we can use that or just remove built-in redis support

Reply:
Removed for now